### DISCUSSION TITLE
Seq user and key file

### DISCUSSION THREAD

**Discussion by emio01tr** [datalust/seq-tickets#2300] [Help] [answered]
Hi,

Which seq file is containing users and api keys information? I'll move our seq storage disk, ı need this information.

![image](https://github.com/user-attachments/assets/180354a1-8227-4591-a0f3-ccb51269141e)

Best Regards.

**Comment by emio01tr** [comment, marked as answer]
I Found it,

Users and api keys information are stored in the Documents folder.

**Reply by nblumhardt** [reply]
That's great - thanks for the follow-up, @emio01tr.
